documentation/content/en/books/arch-handbook/boot/_index.adoc

```diff
@@ -969,29 +965,27 @@ btext calls the routines `recover_bootinfo()`, `identify_cpu()`, `create_pagetab
 |This routine parses the parameters to the kernel passed from the bootstrap. The kernel may have been booted in 3 ways: by the loader, described above, by the old disk boot blocks, or by the old diskless boot procedure. This function determines the booting method, and stores the `struct bootinfo` structure into the kernel memory.
 
 |`identify_cpu`
-|This functions tries to find out what CPU it is running on, storing the value found in a variable `_cpu`.
-
-|`create_pagetables`
-|This function allocates and fills out a Page Table Directory at the top of the kernel memory area.
+|This function tries to find out what CPU it is running on, storing the value found in a variable `_cpu`.
```

Comment:
This is not correct.

Reply:
I think I'd prefer getting a source committer involved in that, because I'm "just" a doc committer.

I'll try to find someone. :)